kmp-nativecoroutines-core/build.gradle.kts

```diff
@@ -30,6 +30,7 @@ kotlin {
         val commonMain by getting {
             dependencies {
                 api(Dependencies.Kotlinx.coroutinesCore)
+                implementation(Dependencies.Kotlinx.atomicfu)
```

Comment:
This could be a `commentTest` dependency

Reply:
Oof duh, I have it correct in the [other PR](https://github.com/rickclephas/KMP-NativeCoroutines/pull/43/files), not sure why I missed it here